Tests/Tests/RZImportTests.m

```diff
@@ -379,6 +380,31 @@ - (void)test_nestedImportCustomKey
     XCTAssertEqualObjects(guysJob.companyName, companyName, @"Failed to import Company Name");
     XCTAssertEqualObjects(guysJob.title, title, @"Failed to import title");
 }
+
+- (void)test_shouldImportValues {
+    //
+    // Testing that it correctly does not import when told not to.
+    //
+    // Testing that if an existing object is supplied, it will still
+    // return that object even if it does not import in values.
+    //
+    // Testing that the didImport method is run on each object
+    //
     
+    NSError *error = nil;
+    NSArray *array = [self loadTestJson:@"test_books" error:&error];
+
+    XCTAssertNil(error, @"Error reading json: %@", error);
```

Comment:
#req this error is defined only if `array` is `nil`, so don't assert it, just log it if `array == nil`.